build.rs

```diff
@@ -20,6 +20,152 @@ fn build_config_name() -> String {
     format!("mkl-dynamic-{}-{}", integer_config, parallelism)
 }
 
+/// Paths required for linking to MKL from MKLROOT folder
+struct MklDirectories {
+    #[allow(dead_code)]
```

Comment:
Why do we need the `allow(dead_code)` annotation...?

Reply:
I removed the annotated field because it is not needed at the moment after your comment